lib/tasks/foreman_theme_satellite_tasks.rake

```diff
@@ -38,9 +28,34 @@ namespace :foreman_theme_satellite do
 
     Rake::Task['rubocop_foreman_theme_satellite'].invoke
   end
+
+  desc 'Validate documentation links against given TOC file'
+  task :validate_docs do
+    toc_file = ENV['TOC']
+    unless toc_file
+      toc_file = File.expand_path('../../test/fixtures/toc.json', __dir__)
+      puts "Using default TOC: #{toc_file}, to override please specify TOC=<toc_file>"
+    end
+
+    require_relative '../../test/link_checker'
+    checker = LinksChecker.new(toc: toc_file)
+
+    require_relative '../foreman_theme_satellite/documentation'
+
+    all_links = ForemanThemeSatellite::Documentation::USER_GUIDE_DICTIONARY
```

Comment:
you should also collect `ForemanThemeSatellite::Documentation::DOCS_GUIDES_LINKS` :)